webapp/src/components/backstage/playbook_runs/playbook_run/context_menu.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import styled from 'styled-components';
+
+import React, {useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {useDispatch} from 'react-redux';
+
+import {showRunActionsModal} from 'src/actions';
+import {exportChannelUrl, finishRun, getSiteUrl} from 'src/client';
+import {TitleButton} from '../../playbook_editor/controls';
+import {PlaybookRun, playbookRunIsActive} from 'src/types/playbook_run';
+import DotMenu, {DropdownMenuItem} from 'src/components/dot_menu';
+import {SemiBoldHeading} from 'src/styles/headings';
+
+import {copyToClipboard} from 'src/utils';
+import {useToasts} from '../../toast_banner';
+import {useAllowChannelExport, useExportLogAvailable} from 'src/hooks';
+import UpgradeModal from '../../upgrade_modal';
+import {AdminNotificationType} from 'src/constants';
+import {outstandingTasks} from 'src/components/modals/update_run_status_modal';
+import {modals} from 'src/webapp_globals';
+import {makeUncontrolledConfirmModalDefinition} from 'src/components/widgets/confirmation_modal';
+
+interface Props {
+    playbookRun: PlaybookRun;
+}
+
+export const ContextMenu = ({playbookRun}: Props) => {
+    const dispatch = useDispatch();
+    const {formatMessage} = useIntl();
+    const {add: addToast} = useToasts();
+
+    const exportAvailable = useExportLogAvailable();
+    const allowChannelExport = useAllowChannelExport();
+    const [showModal, setShowModal] = useState(false);
+
+    const onExportClick = () => {
+        if (!allowChannelExport) {
+            setShowModal(true);
+            return;
+        }
+
+        window.location.href = exportChannelUrl(playbookRun.channel_id);
+    };
+
+    const onFinishRunClick = () => {
+        const outstanding = outstandingTasks(playbookRun.checklists);
+        let confirmationMessage = formatMessage({defaultMessage: 'Are you sure you want to finish the run?'});
+        if (outstanding > 0) {
+            confirmationMessage = formatMessage(
+                {defaultMessage: 'There {outstanding, plural, =1 {is # outstanding task} other {are # outstanding tasks}}. Are you sure you want to finish the run?'},
+                {outstanding});
+        }
```

Comment:
Super nit: move the parenthesis and the colon to a new line
```suggestion
            confirmationMessage = formatMessage(
                {defaultMessage: 'There {outstanding, plural, =1 {is # outstanding task} other {are # outstanding tasks}}. Are you sure you want to finish the run?'},
                {outstanding}
            );
        }
```